ddl/backfilling.go

```diff
@@ -224,21 +241,11 @@ type backfillResult struct {
 	taskID     int
 	addedCount int
 	scanCount  int
+	unsyncErr  error
```

Comment:
No usage?

Reply:
Use it in `waitFinalResult`.

Reply:
`waitFinalResult()` references `backfilWorkerManager.unsyncErr`, not `backfillResult.unsyncErr`.